frontend/src/services/events.js

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios'
+
+// Get request
+const get = async () => {
+    const response = await axios.get('http://localhost:2121/events')
+    return response.data;
+}
+    
+// Post request
+const create = async userData => {
+    const response = await axios.post('http://localhost:2121/events', {
+        ...userData
+    })
```

Comment:
@Caleb-Cohen That looks good. Just a minor nitpick. You can do this instead:

```suggestion
const response = await axios.post('http://localhost:2121/events', userData)
```